src/mongo/tree/MongoDatabaseTreeItem.ts

```diff
@@ -105,6 +105,7 @@ export class MongoDatabaseTreeItem implements IAzureParentTreeItem {
 		if (command.name === 'createCollection') {
 			return reportProgress(this.createCollection(stripQuotes(command.arguments.join(','))).then(() => JSON.stringify({ 'Created': 'Ok' })), 'Creating collection');
 		} else {
+			context.properties["executeInShell"] = "true";
```

Comment:
There's another 'executeCommandInShell' call a few lines up. Make sure to set it there, too

Reply:
Moved telemetry to 'executeCommandInShell' call itself, thanks.